clang/test/CodeGenSYCL/code_location.cpp

```diff
@@ -0,0 +1,101 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem -sycl-std=2020 -fsycl-int-header=%t.h %s -o %t.out
+// RUN: FileCheck -input-file=%t.h %s
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem -sycl-std=2020 -DNDEBUG -fsycl-int-header=%t2.h %s -o %t2.out
+// RUN: FileCheck -input-file=%t2.h %s
+
```

Comment:
Please add a comment here indicating what the test tests for.

Reply:
The test is supposed to test for different outputs in int header, depending on whether NDEBUG flag is specified, however, @elizabethandrews pointed out that perhaps this is not how this mechanism should work. Waiting for requirements clarification from @alexbatashev 
Please bear with me while we wait for this clarification.

Reply:
At the moment, I removed RUN lines and CHECK lines for the NDEBUG case

Reply:
Updated test